src/main/java/uk/gov/hmcts/darts/event/exception/EventError.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.darts.event.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.darts.common.exception.DartsApiError;
+
+@Getter
+@RequiredArgsConstructor
+public enum EventError implements DartsApiError {
+
+    EVENT_DATA_NOT_FOUND(
+          "422",
+          HttpStatus.BAD_REQUEST,
+          "Data on the event could be reconciled with Darts records");
```

Comment:
Should this `title` say **not** reconciled? Or maybe starts with "Failed to reconcile..."?